libs/expo/betterangels/src/lib/screens/Client/Docs/UploadModal/DriverLicense/index.tsx

```diff
@@ -9,7 +9,7 @@ import {
   TextRegular,
```

Comment:
Hey @mikefeldberg 
I saw the text has changed as "Upload CA ID or CA Driver's License" in the Upload page, however, 
I didn't see "CA ID or CA Driver's License" in the Upload Files.

<img src="https://github.com/user-attachments/assets/c7a6debe-e007-4914-95dd-1e82d4b44814" width="360" />

Reply:
that's in a separate ticket https://betterangels.atlassian.net/browse/DEV-828

Reply:
Sorry one more thing I found...

The "Photo ID" in the Upload Files page should be "Other Photo ID (e.g., Out of State)"

<img src="https://github.com/user-attachments/assets/26f2a41f-2652-40c1-ba9d-8163192719ed" width="320" />

Reply:
@shiozakiyuko that's in a separate ticket https://betterangels.atlassian.net/browse/DEV-828

Reply:
Oh my bad! I just saw a ticket that fixes this issue. Thanks so much! Disregard these comments!